docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -31,7 +31,7 @@ In this tutorial, you will:
 
 ## Create a custom functions project
 
-You’ll begin this tutorial by using the Yo Office generator to create the files that you need for your custom functions project.
+You’ll begin this tutorial by using the Yo Office generator to create the files that you need for your custom functions project. If you have previously installed yo office, make sure to update your package to pull the latest from npm. You can do this by running `npm update generator=office`.
```

Comment:
Recommend:

Ensure you have the latest version of Yo Office by running:
npm install -g yo generator-office

The run Yo Office using:
yo office